velox/docs/functions/spark/math.rst

```diff
@@ -101,6 +101,14 @@ Mathematical Functions
     Returns ``x`` rounded down to the nearest integer.
     Supported types are: BIGINT and DOUBLE.
 
+.. spark:function:: hex(x) -> varchar
+
+    Converts ``x`` to hexadecimal.
+    Supported types are: BIGINT, VARBINARY and VARCHAR.
+    If the argument is a BIGINT or binary, hex returns the number as a STRING in hexadecimal format.
+    Otherwise if the number is a STRING, it converts each character into its hex representation
```

Comment:
> the number is a STRING

Do you mean "the argument is VARCHAR"? Would you add some examples?

Reply:
already add examples in doc.